manifests/params.pp

```diff
@@ -131,4 +138,20 @@
   $logstash14_gpgcheck = '1'
   $logstash14_includepkgs = absent
   $logstash14_exclude = absent
+
+  # Nginx Settings.
+  $os_short_name = $::operatingsystem ? {
+    'CentOS' => 'centos',
```

Comment:
It looks to me that the 'rhel' and 'centos' directories are exactly the same, is there a reason we couldn't just always use 'centos'?